cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -49,7 +50,7 @@ public void run() {
             try {
                 yamlFileContent = FileUtils.readLocalFile(commandOptions.getYamlFilePath());
             } catch (IOException e) {
-                System.err.println(e.getMessage());
+                Log.exception(e, "Failed to read YAML file");
```

Comment:
as @felix-oq  mentioned,
I also prefer constants for such Strings.

Reply:
This string is unrelated per se to the logger. The question is where to put such a string.

As mentioned in the chat, the main issue is that every `Runnable` contains the ever same error handling stuff. It would be better to have central error handling instead of having a static const string somewhere.